test/integration/admin/api/objects_controller_test.rb

```diff
@@ -0,0 +1,80 @@
+require 'test_helper'
+
+class Admin::Api::ObjectsControllerTest < ActionDispatch::IntegrationTest
+
+  CONTROLLER = Admin::Api::ObjectsController
+
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+
+    host! @provider.admin_domain
+
+    login_provider @provider
+  end
+
+  def test_status_404
+    get admin_api_objects_status_path(object_type: 'service', object_id: 1)
+    assert_response 404
+  end
+
+  def test_status_200
+    service = FactoryBot.create(:service, account: @provider)
```

Comment:
Just for efficiency:
```suggestion
    service = @provider.default_service
```

Because the factory `provider_account` already creates a very complete service so we don't need to create another one for this scenario :smile: